src/UI/PlayerSetupCommanderController.java

```diff
@@ -33,10 +30,7 @@ public PlayerSetupCommanderController(ArrayList<CommanderInfo> infos, PlayerSetu
     myPlayerInfo = playerInfo;
     shouldSelectMultiCO = tagMode.supportsMultiCmdrSelect;
 
-    HashSet<COSpriteSpec> uniqueFactions = new HashSet<>();
-    for( int co = 0; co < cmdrInfos.size(); ++co )
-      uniqueFactions.add(cmdrInfos.get(co).baseFaction);
-    canonFactions = uniqueFactions.toArray(new COSpriteSpec[0]);
+    var canonFactions = UIUtils.CANON_FACTIONS;
```

Comment:
Is it really so hard to say `CoSpriteSpec[]`? What's the advantage for using `var`?

Reply:
Less typing.
I can make it explicit if you think `UIUtils.CANON_FACTIONS` is too ambiguous re: type.